docs/book/src/SUMMARY.md

```diff
@@ -58,11 +58,12 @@
         - [Multi-tenancy](./developer/architecture/controllers/multi-tenancy.md)
         - [Support multiple instances](./developer/architecture/controllers/support-multiple-instances.md)
     - [Provider Implementers](./developer/providers/implementers.md)
-        - [v1alpha1 to v1alpha2](./developer/providers/v1alpha1-to-v1alpha2.md)
-        - [v1alpha2 to v1alpha3](./developer/providers/v1alpha2-to-v1alpha3.md)
-        - [v1alpha3 to v1alpha4](./developer/providers/v1alpha3-to-v1alpha4.md)
-        - [v1alpha4 to v1beta1](./developer/providers/v1alpha4-to-v1beta1.md)
+        - [v0.1 to v0.2](./developer/providers/v0.1-to-v0.2.md)
+        - [v0.2 to v0.3](./developer/providers/v0.2-to-v0.3.md)
+        - [v0.3 to v0.4](./developer/providers/v0.3-to-v0.4.md)
```

Comment:
Q: Given that we are preserving & linking older versions of the books, should we start dropping old migration documents and keep only the latest 2/3 (for sake of simplicity)?

Reply:
Good point. I think it's fine to only keep the last 2/3 around. I kept everything v0.4+ for now given that we currently don't have links for a v1.0/v1.1 book (pending discussion on the book issue)